src/components/Description/index.tsx

```diff
@@ -1,56 +1,70 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
-import PropTypes from 'prop-types'
-import { WeakValidationMap } from 'react'
-import Box, { BoxProps } from '../Box'
 
-const Term = styled(Box.withComponent('dt'))`
+const Term = styled.dt`
   font-weight: 500;
-  color: ${({ theme }) => theme.colors.neutral.textHover};
+  color: ${({ theme }) => theme.colors.neutral.textStrong};
+  display: inline-block;
   &:after {
     content: ':';
   }
 `
-const Desc = styled(Box.withComponent('dd'))`
+
+const Desc = styled('dd', {
+  shouldForwardProp: prop => !['capitalize'].includes(prop),
+})<{ capitalize?: boolean }>`
+  display: flex;
+  align-items: center;
   color: ${({ theme }) => theme.colors.neutral.text};
-  margin: 0;
-  margin-top: ${({ theme }) => theme.space['1']};
+  margin: 8px 0 0 0;
+  ${({ capitalize }) =>
+    capitalize &&
+    css`
+      text-transform: capitalize;
+    `}
 `
 
-const StyledBox = styled(Box, {
-  shouldForwardProp: prop => !['inline', 'selectable'].includes(prop),
-})<{ inline: boolean; selectable: boolean }>`
+interface DescriptionProps {
+  ellipsis?: boolean
+  inline?: boolean
+  userSelect?: boolean
+  width?: string
+}
+
+const BareDescription = styled('dl', {
+  shouldForwardProp: prop =>
+    !['inline', 'ellipsis', 'userSelect'].includes(prop),
+})<DescriptionProps>`
   font-size: 16px;
   line-height: 16px;
```

Comment:
Possible to use text / theme.typography properties instead of hard coded values ?

Reply:
There is no typography with a line-height 16

Reply:
Maybe we should see with design what kind of typography we can use in this case?

Reply:
@matthprost so process design system, can we leave it as is for this PR and refactor it when the component as been spec-ed ?

Reply:
@chambo-e yes go ahead :)